riscv/riscv.mk.in

```diff
@@ -39,6 +39,7 @@ riscv_precompiled_hdrs = \
 	insn_template.h \
 
 riscv_srcs = \
+    isa_parser.cc \
```

Comment:
This file uses tab indentation (because that's how my `.vimrc` is set up for Makefiles).

Reply:
Gah, that's the second time I've made that mistake. Sorry!